docs/architecture/ADR-002-QGB-Overview.md

```diff
@@ -0,0 +1,75 @@
+# ADR 003: ValSet
+
+## Changelog
+
+- {date}: {changelog}
+
+## Context
+
+> This section contains all the context one needs to understand the current state, and why there is a problem.
+> It should be as succinct as possible and introduce the high level idea behind the solution.
+
+Following the [Quantum Gravity Bridge](https://github.com/celestiaorg/quantum-gravity-bridge) development, we decided to make the orchestrator and relayer part of Celestia-app.
```

Comment:
This is an important statement to discuss, but I think starting off with something more fundamental to why we could do this in the first place would be useful. Rewriting the relayer and the orchestrator was done because our relayer can be designed to be trust minimized, where a two way bridge requires would require each validator to run a trusted relayer. Since we had to rewrite them and they were simplified, we decided to move things to the app. Either in this section or another, we could also compare and contract the consequences of moving to the app.

We might also want to discuss how we are using a similar design to a portion of the normal gravity bridge's application integration. Technically, we don't even need the additional state for the MVP, we only need the state for slashing, but since we are planning on doing slashing in the future, the state changes make it trivial for the relayer to find the data needed to relay to the evm chain.